dCommon/GeneralUtils.h

```diff
@@ -196,9 +181,10 @@ namespace GeneralUtils {
 	 * @returns An std::optional containing the desired value if it is equivalent to the string
 	*/
 	template <std::floating_point T>
-	[[nodiscard]] std::optional<T> TryParse(const std::string_view str) noexcept try {
+	[[nodiscard]] std::optional<T> TryParse(const std::string_view str) noexcept
+	try {
```

Comment:
There isnt much of a point to a function try block outside of a constructor and this syntax imo is less readable than a regular try catch

Reply:
I'm going to strongly disagree here. The only syntactic difference is an extra pair of curly brackets and another level of indentation. In my opinion, that is the more ugly and less readable alternative.

Reply:
Well we have a standard of the previous syntax across dlu and I don't see any benefit here aside from just a different phrasing

Reply:
If you feel it is a blocking issue I will change it, but it isn't something I personally agree with

Reply:
There are plenty disagreements I have with others in regards to coding style, I don't tend to block with style changes are it will be better to just do a format pass to fix and it's mostly syntax changes.  Feel free to keep as is